Makefile

```diff
@@ -316,6 +318,14 @@ shell/public/email-494949.svg: icons/email.svg
 	@$(call color,custom color $<)
 	@sed -e 's/#111111/#494949/g' < $< > $@
 
+shell/public/question-a9a9a9.svg: icons/question.svg
```

Comment:
I think we should be using fonticons for all newly-introduced icons, otherwise we're just creating additional work for ourselves later when we finally switch all the SVG's over.